### DISCUSSION TITLE
How can conversation be clearer?

### DISCUSSION THREAD

**Discussion by jojo12** [slabbi/uddeIM#93] [General]
Hi, first of all 5.4 is very good, I like it.
But: when I get an answer, I don' t know for which question in a serie of questions. I see the date, but not the content. It's not easy to search.
Perhaps you know better communication rules?
Or how can uddeIM facilitate communication?

**Comment by slabbi** [comment]
The Postbox Plugin enables a better message view.

**Comment by joomod** [comment]
yes, I use postbox sine old J3 days :)

but also in classic In/Out view you have a notice line special for your needs

![uddeim-inorg](https://github.com/slabbi/uddeIM/assets/149375504/d6b7c9c8-615e-405a-9497-2a4af87cba48)

I may push up this (added some bs classes)
![uddeim-in](https://github.com/slabbi/uddeIM/assets/149375504/c6908168-f013-44d2-a1ca-345b95accab7)

**Comment by joomod** [comment]
ok, I think the btn design is useful, so I will add this to In/Outbox

(also I will make some small correction in **uddeimlib50.php**, as I have
seen the actual script (checking the groupid will throw an error, if the
site is opened but the user is logged out by the system)

**Comment by joomod** [comment]
@jojo12 
does this fit your needs?

**Comment by jojo12** [comment]
I tried but it seemed not finished. In postbox at the left I had a lightblue square. I dont know what this means?

**Comment by joomod** [comment]
hmm, 
this should not take place in the postbox (just for in and outbox)
can you give me a screenshot

**Comment by jojo12** [comment]
in postbox its away and in and out is good with this blue and the description. Thanks.
Yes, for me it's much more clearer, but I had to learn, to answer always with the reply and not with a new!

**Reply by joomod** [reply]
>  I had to learn, to answer always with the reply and not with a new!
yes, without that no system will know what you are replying to :)

please give me a screenshot
(the buttons depend on the template+bootstrap classes, so they may differ to my system)

**Comment by jojo12** [comment]
![grafik](https://github.com/slabbi/uddeIM/assets/2960236/e12b13b3-a664-4e6f-a017-ebaf47cd007a)
template is new flat, with the todays compiled com_uddeIM_j5

**Comment by joomod** [comment]
ok, I see and will edit.
the btn style should only appear if the link is given, so I will move the btn from the div to the href
